tests/integrationv2/fixtures.py

```diff
@@ -21,21 +21,24 @@ def managed_process():
     """
     processes = []
 
-    def _fn(provider_class: Provider, options: ProviderOptions, timeout=5, send_marker=None, close_marker=None):
+    def _fn(provider_class: Provider, options: ProviderOptions, timeout=5, send_marker=None, close_marker=None, expect_stderr=None):
```

Comment:
everywere else this defaults to `False` is `None` intentional here? `expect_stderr=None`

Reply:
`None` is intentional here. If not set, we default to the provider's `expect_stderr`. Think of this as an optional override.